arch/arm/soc/st_stm32/stm32f7/Kconfig.defconfig.stm32f746xg

```diff
@@ -0,0 +1,28 @@
+# Kconfig - ST STM32F746XG MCU configuration options
+#
+# Copyright (c) 2018 Yurii Hamann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if SOC_STM32F746XG
+
+config SOC
+	string
+	default "stm32f746xx"
+
+if GPIO_STM32
+
+config GPIO_STM32_PORTJ
+	default y
+
+config GPIO_STM32_PORTK
+	default y
+
++endif # GPIO_STM32
```

Comment:
I just realized there is still a small rebase issue, there is a `+` before the `endif`.

Reply:
Thank you. I have updated the code.